metashare/multisalesforce/management/commands/populate_social_apps.py

```diff
@@ -4,15 +4,19 @@
 
 
 class Command(BaseCommand):
-    help = "Populates the required Social Apps."
+    help = "Adds necessary database entries for GitHub OAuth."
 
     def add_arguments(self, parser):
-        parser.add_argument("--prod-id")
-        parser.add_argument("--prod-secret")
-        parser.add_argument("--test-id")
-        parser.add_argument("--test-secret")
```

Comment:
I'm not 100% clear on why we're removing these... Oh, does MS require logging into a production or developer SF org?

Reply:
Because I wanted to add them in incrementally, and see if I can get it not-needing-three-backends, too.

Reply:
(Three backends is gonna make the connect-accounts logic a lot hairier, so I'm gonna try to avoid it.)